data/monthly.csv

```diff
@@ -1,826 +1,2302 @@
-2004-09,405.402
-2004-10,420.210
-2004-11,439.059
-2004-12,442.974
-2005-01,424.080
-2005-02,423.430
-2005-03,434.355
-2005-04,429.140
-2005-05,422.903
-2005-06,430.302
-2005-07,424.745
-2005-08,437.773
-2005-09,455.936
-2005-10,470.107
-2005-11,476.668
-2005-12,509.423
-2006-01,549.433
-2006-02,555.518
-2006-03,557.215
-2006-04,611.853
-2006-05,676.769
-2006-06,597.898
-2006-07,633.093
-2006-08,631.557
-2006-09,600.150
-2006-10,586.648
-2006-11,626.825
-2006-12,629.513
-2007-01,630.352
-2007-02,665.103
-2007-03,655.891
-2007-04,680.008
-2007-05,668.310
-2007-06,655.714
-2007-07,665.266
-2007-08,664.530
-2007-09,710.645
-2007-10,754.480
-2007-11,808.311
-2007-12,803.618
-2008-01,887.784
-2008-02,924.283
-2008-03,971.055
-2008-04,911.600
-2008-05,889.125
-2008-06,889.536
-2008-07,941.167
-2008-08,840.388
-2008-09,824.920
-2008-10,812.815
-2008-11,757.850
-2008-12,819.940
-2009-01,857.726
-2009-02,939.763
-2009-03,925.989
-2009-04,892.663
-2009-05,926.855
-2009-06,947.807
-2009-07,934.272
-2009-08,949.500
-2009-09,996.443
-2009-10,1043.511
-2009-11,1126.119
-2009-12,1135.012
-2010-01,1119.575
-2010-02,1095.800
-2010-03,1115.554
-2010-04,1148.475
-2010-05,1204.321
-2010-06,1232.382
-2010-07,1196.000
-2010-08,1213.464
-2010-09,1271.461
-2010-10,1343.190
-2010-11,1371.784
-2010-12,1393.512
-2011-01,1360.475
-2011-02,1371.313
-2011-03,1422.848
-2011-04,1474.431
-2011-05,1512.188
-2011-06,1528.380
-2011-07,1568.526
-2011-08,1759.500
-2011-09,1780.648
-2011-10,1667.893
-2011-11,1735.977
-2011-12,1652.725
-2012-01,1656.095
-2012-02,1743.095
-2012-03,1675.057
-2012-04,1648.539
-2012-05,1585.114
-2012-06,1595.632
-2012-07,1592.784
-2012-08,1625.682
-2012-09,1741.925
-2012-10,1746.348
-2012-11,1724.352
-2012-12,1687.342
-2013-01,1671.886
-2013-02,1630.688
-2013-03,1591.013
-2013-04,1485.905
-2013-05,1416.143
-2013-06,1342.700
-2013-07,1284.348
-2013-08,1345.048
-2013-09,1348.464
-2013-10,1314.402
-2013-11,1277.417
-2013-12,1221.588
-2014-01,1243.068
-2014-02,1298.713
-2014-03,1336.560
-2014-04,1299.175
-2014-05,1288.913
-2014-06,1277.857
-2014-07,1312.989
-2014-08,1297.005
-2014-09,1241.330
-2014-10,1223.565
-2014-11,1176.413
-2014-12,1200.440
-2015-01,1249.333
-2015-02,1231.100
-2015-03,1180.636
-2015-04,1198.253
-2015-05,1197.684
-2015-06,1182.248
-2015-07,1131.580
-2015-08,1117.525
-2015-09,1124.905
-2015-10,1157.123
-2015-11,1088.388
-2015-12,1068.317
-2016-01,1095.655
-2016-02,1194.893
-2016-03,1246.312
-2016-04,1241.452
-2016-05,1259.755
-2016-06,1273.580
-2016-07,1337.429
-2016-08,1340.861
-2016-09,1326.618
-2016-10,1268.929
-2016-11,1240.293
-2016-12,1152.165
-2017-01,1192.648
-2017-02,1233.390
-2017-03,1231.063
-2017-04,1267.150
-2017-05,1245.250
-2017-06,1261.277
-2017-07,1235.100
-2017-08,1281.720
-2017-09,1317.045
-2017-10,1280.677
-2017-11,1283.189
-2017-12,1265.674
-2018-01,1332.809
-2018-02,1333.775
-2018-03,1325.562
-2018-04,1335.332
-2018-05,1303.618
-2018-06,1282.126
-2018-07,1238.064
-2018-08,1201.859
-2018-09,1199.198
+Date,Gold
```

Comment:
@gradedSystem second column name should be `Price`

Reply:
Updated